Scrum 42 wrc dataset

hashemJaber opened the pull request:
- Added new features related to:
                                                      -  Historical fire trends of areas 
                                                      -  Risk to structures  
                                                      -  Probability of fire of reaching type 1 (exceeding 4ft)
                                                      -  Probability of fire of reaching type 2 (exceeding 8ft)
                                                      -  Conditional_flame_length
                                                      -  Wildfire_hazard_potential
- Added base skeleton tools for feature processing and data cleaning:
                                                      -  Method 1: method fills the missing days and etc missing vlaues with the average of the day before and the day after.
                                                      -  Method 2:  method takes in a dataframe, extracts date and time and then hot encodes the day, hour, and month.

EdotAnt committed:
extracted and analyzed drought and fire condition data

Shawn-Kim96 committed:
feat: add processed data for landfire

Shawn-Kim96 committed:
feat: add notebooks on analyzing and adding landfire data

Shawn-Kim96 committed:
feat: update  python dependencies

hashemJaber committed:
SCRUM-42-tools

hashemJaber committed:
SCRUM 43&44

- Added tools for cleaning and preprocessing data (functional only)
  -   Added new features on risk to structures and meta historical risk data

Shawn-Kim96 commented:
@hashemJaber I can't find scripts / notebook files. Can you check if you uploaded it?
It'll be nice if you add notebook and a scripts. Also, add some documents (README.md) or write some process on notebook too!

hashemJaber committed:
Added new features related to:
- Historical fire trends of areas
- Risk to structures
- Probability of fire of reaching type 1 (exceeding 4ft)
- Probability of fire of reaching type 2 (exceeding 8ft)
- Conditional_flame_length
- Wildfire_hazard_potential
Added base skeleton tools for feature processing and data cleaning:
- Method 1: method fills the missing days and etc missing vlaues with the average of the day before and the day after.
- Method 2: method takes in a dataframe, extracts date and time and then hot encodes the day, hour, and month.

Shawn-Kim96 commented:
@hashemJaber  I just checked your code. I think you need to make one more script that does
get an input of data
script addes column to the data

This is because your notebook and data is currently adding columns to given fire data. However, we have to add additional data to non-wildfire data (Rodrigo data), so it will be good to make a script. You can just take a look at my [script](https://github.com/Shawn-Kim96/California_Wild_Fire_Prediction/pull/5/files#diff-eaadaa45425b06f697178ce0ef24199d55ec2f12fcea48c843fc20a037ba8d82). 
After the script, @KenKaiZhang  can execute both scripts from total data (fire data + non-wildfire data), then we will have our final data

Shawn-Kim96 commented:
Change minimum number of review approval from 2 to 1.
